readme.md

```diff
@@ -213,36 +249,46 @@ In our `app/views/artists/index.html.erb`
 
 Now when we visit `/artists` in the browser, we see a list (index) of all artists. We have included links to the new page, and for the show page, but we haven't created those yet so....
 
-### You-Do: Show and New Actions (15/70)
-- Define `show`, and `new` controller actions for artists
-- Create `show`, and `new` views for artists
-- Display relevant info for an artist when you visit their show page
-- When you visit the `new` page, the relevant form should be displayed
-- Don't worry about submitting the form just yet
+### You-Do: Show and New Actions (15 / 85)
 
-## BREAK (10/80)
+- Define `show`, and `new` controller actions for `artists`
+- Create `show`, and `new` views for `artists`
+- Display relevant info for an `artist` when you visit their `show` page
+- When you visit the `new` page, you should see a header with `New Artist` as text for your html
 
-## WE-DO: Artists Create Action (20/100)
-So at this point we have a way to see all artists, view info about a specific artist, as well as see a form to add a new artist.
+If you finish early:
+- Research Rails `form for` helper method and think about how you would include a form to get user input necessary to create a new artist instance and have it persist . What happens within Rails when we hit submit on the form?
 
-```rb
-<form action="/artists" method="post" >
-  <label for="name">Name:</label>
-  <input name="name">
+## WE-DO: Artists Create Action (20 / 105)
+
+So at this point we have a way to see all artists, view info about a specific artist. Let's continue on by adding a form for the user to add their own artists to our application.
 
-  <label for="photo_url">Photo URL:</label>
-  <input name="photo_url">
+In Rails, we have access to a variety of helper methods to erase the pain of having to writing repetitive boilerplate `html` code.
 
-  <label for="nationality">Nationality:</label>
-  <input name="nationality">
+This seems like a great time to utilize Rails `form_for` [helper method](http://guides.rubyonrails.org/form_helpers.html#dealing-with-model-objects):
 
-  <input type="submit" value="Create">
-</form>
 ```
+<%= form_for @artist do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+
+  <%= f.label :nationality %>
+  <%= f.text_field :nationality %>
+
+  <%= f.label :photo_url %>
+  <%= f.text_field :photo_url %>
+
+  <%= f.submit "Create" %>
+<% end %>
+```
+
+Here, we are essentially saying we want a form that will result in the creation of a new instance, in this case `@artist`. In the form body, we just have to make sure that our fields match our Model's attributes and we should be good to go.
 
-But what happens when we try to submit this form?
+Basically, this helper with compile down to html, and the Rails will fill in the appropriate information particular to our app.
 
```

Comment:
I assume you will, but show them the source code for this. That way they can map what they see in the helper to what they see in the HTML.